src/core/control/Control.cpp

```diff
@@ -2132,30 +2132,43 @@ auto Control::openFile(fs::path filepath, int scrollToPage, bool forceOpen) -> b
-                FS(_F("The attached background file {1} could not be found. It might have been moved, renamed or "
-                      "deleted.\nIt was last seen at: {2}") %
-                   missingFilePath.filename().string() % missingFilePath.parent_path().string());
-        const std::string msg2 =
-                FS(_F("The background file {1} could not be found. It might have been moved, renamed or deleted.\nIt "
-                      "was last seen at: {2}") %
-                   missingFilePath.filename().string() % missingFilePath.parent_path().string());
-        GtkWidget* dialog =
-                gtk_message_dialog_new(getGtkWindow(), GTK_DIALOG_MODAL, GTK_MESSAGE_QUESTION, GTK_BUTTONS_NONE, "%s",
-                                       loadHandler.isAttachedPdfMissing() ? msg1.c_str() : msg2.c_str());
 
-        gtk_dialog_add_button(GTK_DIALOG(dialog), _("Select another PDF"), 1);
-        gtk_dialog_add_button(GTK_DIALOG(dialog), _("Remove PDF Background"), 2);
-        gtk_dialog_add_button(GTK_DIALOG(dialog), _("Cancel"), 3);
+        std::string msg;
+
+        if (loadHandler.isAttachedPdfMissing()) {
+            msg = FS(_F("The attached background file \"{1}\" could not be found. It might have been moved, "
+                        "renamed or "
+                        "deleted.\nIt was last seen at: \"{2}\"") %
+                     missingFilePath.filename().string() % missingFilePath.parent_path().string());
+        } else {
+            msg = FS(_F("The background file \"{1}\" could not be found. It might have been moved, renamed or "
+                        "deleted.\nIt "
+                        "was last seen at: \"{2}\"") %
+                     missingFilePath.filename().string() % missingFilePath.parent_path().string());
+        }
+
+        // try to find file in current directory
+        auto proposedPdfFilepath = filepath.parent_path() / missingFilePath.filename();
+        bool proposedPdfFileExits = fs::exists(proposedPdfFilepath);
+        if (proposedPdfFileExits) {
+            msg += FS(_F("\nProposed replacement file: \"{1}\"") % proposedPdfFilepath.string());
+        }
+        GtkWidget* dialog = gtk_message_dialog_new(getGtkWindow(), GTK_DIALOG_MODAL, GTK_MESSAGE_QUESTION,
+                                                   GTK_BUTTONS_NONE, "%s", msg.c_str());
+
+        if (proposedPdfFileExits) gtk_dialog_add_button(GTK_DIALOG(dialog), _("Use proposed PDF"), 1);
+        gtk_dialog_add_button(GTK_DIALOG(dialog), _("Select another PDF"), 2);
+        gtk_dialog_add_button(GTK_DIALOG(dialog), _("Remove PDF Background"), 3);
+        gtk_dialog_add_button(GTK_DIALOG(dialog), _("Cancel"), 4);
         gtk_window_set_transient_for(GTK_WINDOW(dialog), GTK_WINDOW(this->getWindow()->getWindow()));
         int res = gtk_dialog_run(GTK_DIALOG(dialog));
         gtk_widget_destroy(dialog);
 
-        if (res == 2)  // remove PDF background
+        if (res == 3)  // remove PDF background
```

Comment:
For readability, prefer a switch statement here:
```suggestion
        switch(res) {
        case REMOVE_BACKGROUND:
```